testcases/gcov_setup.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+# OpenPOWER Automated Test Project
+#
+# Contributors Listed Below - COPYRIGHT 2017
+# [+] International Business Machines Corp.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+'''
+gcov
+----
+
+A "Test Case" for extracting GCOV code coverage data from skiboot. The real
+use of this test case and code is to help construct code coverage reports
+for skiboot.
+'''
+
+import os
+import tempfile
+import unittest
+
+import OpTestConfiguration
+from common.OpTestSystem import OpSystemState
+from common.Exceptions import CommandFailed
+from common import OpTestInstallUtil
+from common.OpTestUtil import OpTestUtil
+
+import logging
+import OpTestLogger
+log = OpTestLogger.optest_logger_glob.get_logger(__name__)
+
+
+class gcov(unittest.TestCase):
```

Comment:
class name should start with caps letter,  and same as py file name most of the time